build.sh

```diff
@@ -21,11 +34,68 @@ patch -d /usr/lib/live/build/ < live-build-fix-syslinux.patch
 # https://salsa.debian.org/installer-team/debootstrap/blob/master/debian/changelog
 ln -sfn /usr/share/debootstrap/scripts/gutsy /usr/share/debootstrap/scripts/focal
 
-echo -e "
-#----------------------#
-# RUN TERRAFORM SCRIPT #
-#----------------------#
+build () {
+  BUILD_ARCH="$1"
+
+  mkdir -p "$BASE_DIR/tmp/$BUILD_ARCH"
+  cd "$BASE_DIR/tmp/$BUILD_ARCH" || exit
+
+  # remove old configs and copy over new
+  rm -rf config auto
+  cp -r "$BASE_DIR"/etc/* .
+  # Make sure conffile specified as arg has correct name
+  cp -f "$BASE_DIR"/"$CONFIG_FILE" terraform.conf
+
+  # Symlink chosen package lists to where live-build will find them
+  ln -s "package-lists.$PACKAGE_LISTS_SUFFIX" "config/package-lists"
+  # symlink appcenter archive
+  if [ "$INCLUDE_APPCENTER" = "yes" ]; then
+    ln -s "appcenter/appcenter.list.binary" "archives/appcenter.list.binary"
+    ln -s "appcenter/appcenter.key.binary" "archives/appcenter.key.binary"
+  fi
+
+  echo -e "
+#------------------#
+# LIVE-BUILD CLEAN #
+#------------------#
 "
+  lb clean
+
+  echo -e "
+#-------------------#
+# LIVE-BUILD CONFIG #
+#-------------------#
+"
+  lb config
+
+  echo -e "
+#------------------#
+# LIVE-BUILD BUILD #
+#------------------#
+"
+  lb build
+
+  echo -e "
+#---------------------------#
+# MOVE OUTPUT TO BUILDS DIR #
+#---------------------------#
+"
+
+  YYYYMMDD="$(date +%Y%m%d)"
+  OUTPUT_DIR="$BASE_DIR/builds/$BUILD_ARCH"
+  # remove old builds and move in new
+  rm -rf "$BASE_DIR"/builds
```

Comment:
We also can't delete `builds` inside the function here as the function is called for each arch. So we'd be deleting the other arches before being able to upload them.